docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :release:`1.33.0 <2024-05-08>`
+* :bug:`7915` - Fixed an issue where frozen balances were not displayed in OKX accounts. With this update, all balances, including frozen ones, are now correctly shown together, enhancing visibility and account management for users.
```

Comment:
Simplify and put above the 1.33.0. That release has already happened. Write as I show below.

```suggestion
* :bug:`7915` - Show OKX balances locked in active trades.

* :release:`1.33.0 <2024-05-08>`
```